pkg/accounting/rollup/rollup.go

```diff
@@ -109,5 +109,15 @@ func (r *Rollup) Query(ctx context.Context) error {
 		r.logger.Info("Rollup only found tallies for today")
 		return nil
 	}
-	return Error.Wrap(r.db.SaveRollup(ctx, latestTally, rollupStats))
+	err = r.db.SaveRollup(ctx, latestTally, rollupStats)
+	if err != nil {
+		return Error.Wrap(err)
+	}
+	//removed the rolled-up raws
+	var rolledUpRawsHaveBeenSaved bool
+	//todo: write files to disk or whatever we decide to do here
```

Comment:
same question as on the other PR: is there a jira ticket for this work?

Reply:
https://storjlabs.atlassian.net/browse/V3-1306